HEL-150 | Browsing collections is challenging with screen reader

tuovinensanttu opened the pull request:
Due the missing translations on collection titles, `lang="fi"` attribute was added to all collections. It will force screen reader to read titles in finnish.

Manual testing instructions:
Using screen reader
1. Navigate to home page -> showcase collection titles should be read in finnish.
2. Navigate to "Explore albums" page -> title should be read in finnish.
3. Navigate to "Own albums" -> again titles should be read in finnish. When user creates favourite album the first time, its language is determined by users current language at the time. Favourites album should be read with the users current language.

Jokinen reviewed (commented):
There were no collections on the frontpage, so I was not able to test the first step.

While testing the second step, I noticed that visual users see the image, see the title and see the owner of the image as they see the logo of the museum. Screen reader users only hear the title and do not hear the owner.

I tested the "My Albums" page. I created an album by favouriting images. In this scenario the album "Favourites" was assigned to me. I noticed two problems. The one that's pertinent to this PR is that although the title was in English ("Favourites"), it was still read in Finnish. This is quite a difficult problem to resolve as the user can edit the title and the description and may be a wontfix.

The other issue was that the album link was not reachable with tab. I could reach it by using <kbd>ctrl</kbd> + <kbd>opt</kbd> + <kbd>right arrow</kbd> while using a screen reader. But the album is not accessible for keyboard users.

tuovinensanttu commented:
Users favourite album should now be read with users current language, I added this note to original message as well. As Jaakko mentioned, albums owner was ignored by screen readers, this is now fixed as well.

Relating to issue where albums might not be targeted with tab, I created a new ticket [HEL-252].(https://helsinkisolutionoffice.atlassian.net/browse/HEL-252)

Jokinen reviewed (approved):
Checked the changes and they work +1
